src/test/issueTests.ts

```diff
@@ -725,6 +728,40 @@ export const issueTests: {
         );
     },
 
+    gh2024() {
```

Comment:
Probably going to pull this out into options.test.ts - these tests are really for checking things which depend on conversion, otherwise we're doing more work than necessary.